app/views/courses/_course.html.erb

```diff
@@ -14,7 +14,18 @@
           <h2 id="<%= course.title.parameterize %>" class="font-medium text-xl md:text-left text-gray-900 dark:text-gray-200">
             <%= course.title %>
           </h2>
-          <span class=" text-sm text-gray-500"><%= course.lessons.size %> Lessons</span>
+          <div class="flex space-x-3 mt-2 sm:mt-1.5 sm:justify-center text-gray-500 dark:text-gray-400">
```

Comment:
Nit: From testing, it looks like we don't need the sm: breakpoint on justify-center

```suggestion
          <div class="flex space-x-3 mt-2 sm:mt-1.5 justify-center text-gray-500 dark:text-gray-400">
```